### ISSUE TITLE
AllowVars, EncryptVar, and EBLSecret position

### ISSUE THREAD

**Issue by moorereason** [cu-library/ezproxy-config-lint#16]
The [AllowVars](https://help.oclc.org/Library_Management/EZproxy/Configure_resources/AllowVars), [EncryptVar](https://help.oclc.org/Library_Management/EZproxy/Configure_resources/EncryptVar), and [EBLSecret](https://help.oclc.org/Library_Management/EZproxy/Configure_resources/EBLSecret) documentation show putting these directives before `URL`:

```
Title Ebook Library
AllowVars u
EncryptVar u astringyoupick
EBLSecret D34A9B3C28F1
URL http://www.yourlib.elib.com/EBLWeb/patron/
DJ elib.com
```

[v0.0.7](https://github.com/cu-library/ezproxy-config-lint/tree/v0.0.7) reports the following:

```
test.txt:5: URL http://www.yourlib.elib.com/EBLWeb/patron/ ← URL directive is out of order
```

It's not clear to me if these directives will work after the `URL` directive.  I'll see if I can test that somehow.

**Comment by moorereason** [comment]
BTW, lots of issues found in the official OCLC stanzas mirrored at https://github.com/kent-state-university-libraries/oclc-ezproxy-database-stanzas.  A great source of test stanzas for this project.

**Comment by cudevmaxwell** [comment]
Should be fixed in [0.0.11](https://github.com/cu-library/ezproxy-config-lint/releases/tag/v0.0.11). That Kent State repo is a great resource, thanks!
